templates/layout/page-footer.html.twig

```diff
@@ -1,3 +1,23 @@
+{# Prepare footer_information attributes. #}
+{% set footer_information_attributes = create_attribute() %}
+
+{# Add footer information classes. #}
+{%
+  set footer_information_classes = [
+    'footer-information',
+    footer_information_style ? footer_information_style,
+  ]
+%}
+{% if page.footer_information %}
+  <section{{ footer_information_attributes.addClass(footer_information_classes).setAttribute('role', 'footer information') }}>
```

Comment:
Please remove the role `footer information`.

It does not match the aria role. 
Refer to https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles

Reply:
okay